tests/cbtest/gtest/test_zrepl_prot.cc

```diff
@@ -1226,7 +1226,15 @@ TEST(Misc, ZreplCheckpointInterval) {
 
 	write_data_and_verify_resp(datasock_slow.fd(), ioseq, 0, 888);
 	write_data_and_verify_resp(datasock_fast.fd(), ioseq, 0, 888);
-	sleep(2);
+	sleep(30);
```

Comment:
ok. I have added it since io_seq was not getting updated in ZAP. But as we discussed this can be reduced since on updating `update_interval`, timer thread will update sleep time. Thanks.